props/saber_sabersense_buttons.h

```diff
@@ -0,0 +1,1046 @@
+      case EVENTID(BUTTON_AUX2, EVENT_PRESSED, MODE_ON):
+        if (accel_.x < -0.15) {
+          pointing_down_ = true;
+        } else {
+          pointing_down_ = false;
+      }
+    return true;
+
+  //  Gesture Controls
+#ifdef SABERSENSE_SWING_ON
+    case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_OFF):
+      // Due to motion chip startup on boot creating false ignition we delay Swing On at boot for 3000ms
+      if (millis() > 3000) {
+        FastOn();
+        //  On();
+      }
+    return true;
+#endif
+
+#ifdef SABERSENSE_TWIST_ON
+    case EVENTID(BUTTON_NONE, EVENT_TWIST, MODE_OFF):
+      // Delay twist events to prevent false trigger from over twisting
+      if (millis() - last_twist_ > 2000 &&
+        millis() - saber_off_time_ > 1000) {
+          FastOn();
+          //  On();
+         last_twist_ = millis();
+        }
+      return true;
+#endif
+
+#ifdef SABERSENSE_TWIST_OFF
+    case EVENTID(BUTTON_NONE, EVENT_TWIST, MODE_ON):
+      // Delay twist events to prevent false trigger from over twisting
+      if (millis() - last_twist_ > 3000) {
+        Off();
+          last_twist_ = millis();
+          saber_off_time_ = millis();
+          battle_mode_ = false;
+        }
+      return true;
+#endif
+
+#ifdef SABERSENSE_STAB_ON
+    case EVENTID(BUTTON_NONE, EVENT_STAB, MODE_OFF):
+      if (millis() - saber_off_time_ > 1000) {
+        FastOn();
+        //  On();
+      }
+    return true;
+#endif
+
+#ifdef SABERSENSE_THRUST_ON
+    case EVENTID(BUTTON_NONE, EVENT_THRUST, MODE_OFF):
+      if (millis() - saber_off_time_ > 1000) {
+        FastOn();
+        //  On();
+      }
+    return true;
+#endif
+
+  //  Multiple Skips only available with 2 button installs.
+  //  Skips forward five fonts if pointing up, skips back five fonts if pointing down.
+    case EVENTID(BUTTON_AUX, EVENT_SECOND_SAVED_CLICK_SHORT, MODE_OFF):
+      // backwards if pointing down
+      SetPreset(current_preset_.preset_num + (fusor.angle1() < -M_PI / 4 ? -5 : 5), true);
+    #ifdef SAVE_PRESET
+      SaveState(current_preset_.preset_num);
+    #endif
+    return true;
+
+  // Skips forward ten fonts if pointing up, skips back ten fonts if pointing down.
+    case EVENTID(BUTTON_AUX, EVENT_THIRD_SAVED_CLICK_SHORT, MODE_OFF):
+      // backwards if pointing down
+        SetPreset(current_preset_.preset_num + (fusor.angle1() < -M_PI / 4 ? -10 : 10), true);
+      #ifdef SAVE_PRESET
+        SaveState(current_preset_.preset_num);
+      #endif
+    return true;
+
+// Saber ON AND Volume Adjust.
+    case EVENTID(BUTTON_POWER, EVENT_FIRST_SAVED_CLICK_SHORT, MODE_OFF):
+      IgnoreClash(50);  //  Hopefully prevents false clashes due to 'clicky' button.
+                        //  Low threshold so as not to conflict with 1-button lockup or volume menu.
+        if (!mode_volume_) {
+          On();
+        } else {
+          if (fusor.angle1() > 0) {
+            VolumeUp();
+          } else {
+            VolumeDown();
+        }
```

Comment:
indent doesn't match

Reply:
It does now, hopefully in all instances..